### DISCUSSION TITLE
Name Entity Recognition using Spacy in Python

### DISCUSSION THREAD

**Discussion by bezantoop** [explosion/spaCy#7654] [Help: Coding & Implementations]
am trying to do Name entity recognition model using spacy in python and i want to take my training data by imorting from file instade of writing in code section like: 
TRAIN_DATA =[
    ("Who is Shaka Khan?", {"entities": [(7, 17, "PERSON")]}),
    ("I like London and Berlin.", {"entities": [(7, 13, "LOC"), (18, 24, "LOC")]}),

when i trying to importing this  as a file through 
file = open("c:\\user\\ani..\\desktop\\train.txt", 'r')
TRAIN_DATA=file.read()

it produce an error which says 
ValueError : not enough value to unpack.

my Environment:
- Operating System: windows 10
- Python Version Used: 3.7
- spaCy Version Used: 2.3.2
- Environment Information:

**Comment by svlandeg** [comment]
Hi! This type of questions are well suited for the discussions forum, so I'll move your question there. This issue will be closed, but will link to the new, open thread.

**Comment by svlandeg** [comment]
Can you share (part of) the contents of the file you're using, and a full code snippet that runs into the error? I'm guessing that it's just a matter of making sure the text is parsed correctly from the file, into a list of examples, where each example is a proper tuple.
